lib/oxrdf/src/literal.rs

```diff
@@ -159,6 +161,63 @@ impl fmt::Display for Literal {
     }
 }
 
+#[cfg(feature = "serde")]
+impl Serialize for Literal {
+    #[inline]
+    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        let mut map: <S as Serializer>::SerializeStruct;
+        match self {
+            Self(LiteralContent::String(value)) => {
+                map = serializer.serialize_struct("Literal", 1)?;
```

Comment:
Nit: I think you can factorize this line

Reply:
Not easily because this has `1` field and the others have `2`.